src/lib/app-helper.js

```diff
@@ -305,6 +309,50 @@ function servicesToGeneratorInput (services) {
   return services.map(s => s.code).filter(s => s).join(',')
 }
 
+/**
+ * Log a warning when overwriting services in the Production Workspace
+ *
+ * @param {string} projectName project name, needed for warning message
+ * @param {string} workspaceName workspace name
+ */
+function warnIfOverwriteServicesInProductionWorkspace (projectName, workspaceName) {
+  if (workspaceName === 'Production') {
+    console.error(chalk.bold(chalk.yellow(
+      `⚠ Warning: you are authorizing to overwrite Services in your *Production* Workspace in Project '${projectName}'.` +
+      `${EOL}This may break any Applications that currently uses existing Service subscriptions in this Production Workspace.`
```

Comment:
grammar: This may break any Applications that currently use existing Service subscriptions in this Production Workspace.